#Panel Identification
std_msgs/String panel_location



#Button Status
std_msgs/Bool btn1
std_msgs/Bool btn2
std_msgs/Bool btn3
std_msgs/Bool btn4
std_msgs/Bool btn5
std_msgs/Bool btn6
std_msgs/Bool btn7
std_msgs/Bool btn8
std_msgs/Bool btn9
std_msgs/Bool btn10
